packages/react/src/hooks/utils/index.ts

```diff
@@ -1,3 +1,4 @@
+export { parseContractResult } from './parseContractResult'
```

Comment:
Let's move this function to `src/utils` rather than `src/hooks/utils`.

Reply:
Oops, didn't realise!